daal4py/sklearn/linear_model/tests/test_ridge.py

```diff
@@ -0,0 +1,51 @@
+# ==============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy
+import pytest
+from sklearn.datasets import make_regression
+
+
+def test_multivariate_ridge_coefficients():
```

Comment:
I would sugget you to have some common Func:
```python
def _test_multivariate_ridge_coefficients(module, random_state):
   ...


def test_multivariate_ridge_coefficients():
    from daal4py.sklearn.linear_model._ridge import Ridge
    random_state = 0
     _test_multivariate_ridge_coefficients(Ridge, random_state)
``` 

Just import and reuse `_test_multivariate_ridge_coefficients` in tests for sklearnex Ridge #1843 
In `sklearnex/preview/linear_model/tests/test_ridge.py` just add:

```python
def test_multivariate_ridge_coefficients():
    from sklearnex.preview.linear_model import Ridge
    random_state = 0
     _test_multivariate_ridge_coefficients(Ridge, random_state)
```